dask/dataframe/core.py

```diff
@@ -1741,6 +1741,13 @@ def __repr__(self):
                                           name=key_split(self._name),
                                           task=len(self.dask))
 
+    @derived_from(pd.Series)
+    def rename(self, index=None, inplace=False):
+        if not inplace:
+            self = self.copy()
+        self.name = index
```

Comment:
This may need some validation. `Series.rename` confusingly has two modes:

1. for Callables / mappings, the index *labels* are changed according to the callable / mapping
2. For scalars, `self.name` is set

https://github.com/pandas-dev/pandas/blob/316acbf065f5d1e30fdc7fcced3a53a2f4b14655/pandas/core/series.py#L2681

Reply:
Ah, good catch. Should be fixed now.